### ISSUE TITLE
Suggestion: When a screen reader such as NVDA is being used, let the tree-view in the main window be easily  focusable

### ISSUE THREAD

**Issue by marrie** [rogerclarkmelbourne/radioddity_gd-77_cps#78]
Situation:

When a screen reader such as NVDA is being used, the tree view cannot be focused easily. You can get NVDA for free at http://nvda-project.org.

Basically,  you   need to use the mouse keys the screen reader has 
to gain focus on the tree-view., which is fine, but not intuitive for some of us. 

I suggest the tab key  to gain easier focus. Also I tell developers to every so often run the software using the screen reader, no screen to make sure the app is accessible.  If you have questions please ask.
